### ISSUE TITLE
BaseFeature is not Supported - Body copy to sketch points Error

### ISSUE THREAD

**Issue by Benjin454** [jmaruska/Duplicate-to-Points#2]
Let me know if there is more information I can provide to help fix this. 
![screen shot](https://user-images.githubusercontent.com/10505629/46696230-4ae92280-cbd7-11e8-9841-c31e0deb75e0.png)

**Comment by jmaruska** [comment]
Yes. Currently this script only works if you’re capturing design history.
